src/SignalR/common/Protocols.Json/src/Protocol/JsonHubProtocol.cs

```diff
@@ -773,7 +773,7 @@ internal static JsonSerializerOptions CreateDefaultSerializerSettings()
                 WriteIndented = false,
                 ReadCommentHandling = JsonCommentHandling.Disallow,
                 AllowTrailingCommas = false,
-                DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingDefault,
+                DefaultIgnoreCondition = JsonIgnoreCondition.Never,
```

Comment:
Breaking change?

Reply:
No, this is correcting a break from earlier. https://github.com/dotnet/aspnetcore/pull/22076/files#diff-81f0f21fef0bc37dc6a70b52a0c3379aR776